Mathlib/Data/Nat/ModEq.lean

```diff
@@ -8,6 +8,9 @@ import Mathlib.Data.Int.Order.Lemmas
 import Mathlib.Logic.Basic
 import Mathlib.Tactic.NormNum
 import Mathlib.Tactic.GCongr.Core
+import Mathlib.Data.Nat.GCD.BigOperators
+import Mathlib.Data.List.MinMax
+import Mathlib.Data.List.FinRange
```

Comment:
You are adding a bunch of imports to a low-level file, which is not good. Please move the results that are about List/Multiset/Finset to a new file, e.g. `Data/Nat/ChineseRemainder`.

bors r-

Reply:
bors d-

Reply:
Yes good point, I have done that